complaint_search/tests/test_es_interface.py

```diff
@@ -1493,3 +1494,89 @@ def test_states__valid(self, mock_search):
         self.assertEqual(mock_search.call_args[1]['doc_type'], 'DOC_TYPE')
         self.assertEqual(mock_search.call_args[1]['index'], 'INDEX')
         self.assertEqual('OK', res)
+
+
+class EsInterfaceTest_Trends(TestCase):
```

Comment:
Because of the upcoming merge with `https://github.com/JeffreyMFarley/ccdb5-api/tree/event-tag`. Could you put this Test Class in its own file?  This will avoid some conflicts and also keep from overstuffing this one file